app/src/apps/osfi.py

```diff
@@ -45,24 +64,81 @@ def get_pie(data, column):
                 width=12,
             )
         ),
+        dbc.Row(
+            dbc.Col(
+                build_figure_container(
+                    title="Évolution temporelles des émissions (électricité)", id="electricity_time_series", footer="Explications..."
+                ),
+                width=12,
+            )
+        ),
+        dbc.Row(
+            dbc.Col(
+                build_figure_container(
+                    title="Évolution temporelles des émissions (gaz)", id="gaz_time_series", footer="Explications..."
+                ),
+                width=12,
+            )
+        ),
     ]
 )
 
 
 @app.callback(
     [
         Output("osfi-all-data-table", "columns"),
+        Output("osfi-all-data-table", "row_selectable"),
+        Output("osfi-all-data-table", "style_data"),
+        Output("osfi-all-data-table", "style_cell_conditional"),
```

Comment:
This one should probably be added in the constructor of the `build_table_container` method as their values aren't dependent of the input of this function.